### ISSUE TITLE
Should not lint on method and data calls on box-imported R6 class.

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#27]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  path/to/module_r6[SomeClass],
)

s <- SomeClass$new()
s$char_attribute
s$method()
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Lint result

```
<text>:8:1: warning: [box_usage_linter] <package/module>$function does not exist.
s$method()
^~~~~~~~
```

## Related issue
- #22 
- #23 
- #25 

## Expected result

Should not lint on R6 method called from a `box` imported R6 class.
